src/Calypso-SystemTools-QueryBrowser/ClyTextEditor.extension.st

```diff
@@ -74,14 +75,13 @@ ClyTextEditor >> referencesToIt [
 
 { #category : #'*Calypso-SystemTools-QueryBrowser' }
 ClyTextEditor >> sendersOf: selectedSelector [
-	
+	selectedSelector isCharacter ifTrue: [ ^ self ].
```

Comment:
Senders should normally work for any literals. We should just protect the check for class reference:
```Smalltalk
ClyBrowserMorph>>browseReferencesTo: aSymbol from: anEnvironment	
	...
	(aSymbol isSymbol and: [aSymbol first isUppercase]) ifTrue: [ 

```
#isSymbol will solve the issue